xbmc/addons/kodi-addon-dev-kit/src/api2/kodi-addon-sharedlibrary/CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+project(kodi-addon-sharedlibrary-api2)
+
+cmake_minimum_required(VERSION 3.0.0)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR})
+
+enable_language(C CXX)
+
+include(../dev-kit-api-level.cmake)
+include(../../../UseMultiArch.cmake)
+
+set(kodi-addon-sharedlibrary_NAME kodi-addon-sharedlibrary)
+set(kodi-addon-sharedlibrary_DESCRIPTION "Kodi add-on API 3 development kit for shared libraries")
+set(kodi-addon-sharedlibrary_API_LEVEL=${ADDON_API_LEVEL})
+set(kodi-addon-sharedlibrary_VERSION ${ADDON_API_VERSION})
+
+add_definitions(-DUSE_DEMUX)
```

Comment:
please always use the target specific versions. add_definitions, include_directories and so on will always propagate into subdirs. usually we don't want that. better be explicit: target_compile_definitions, target_include_directories